laravel/resources/views/administration/fonts.blade.php

```diff
@@ -92,12 +92,9 @@
 
 @foreach($fonts as $fontForEdit)
     @include('modals.fonts.modal-edit', ['fontForEdit'=>$fontForEdit])
```

Comment:
[SUGGESTION]

Only insert the modal edit code, if the user has the permission to access this feature at all:
```suggestion
    @can('edit fonts')
    @include('modals.fonts.modal-edit', ['fontForEdit'=>$fontForEdit])
    @endcan

```